File: languages/java/testData/collision/field.java
package testData.collision;

public class MyClass {
    public int myField = 0;
    public static int collision = 1;

    public void setField(int boi) {
        myField = boi;
    }

    public static void main(String[] args) {
        MyClass instance = new MyClass();
        instance.myFie<caret>ld = 1;
    }
}

File: languages/kotlin/testData/collision/field.kt
package collision

class MyClass(not_collision: Boolean, val collision1: Int = 0) {
    var myField = 0
    private val collision2 = 1
    fun setField(boi: Int) {
        myField = boi
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val instance = MyClass(true)
            instance.myFie<caret>ld = 1
        }
    }
}